apps/github-cascading-app/src/cascading/cascading.spec.ts

```diff
@@ -198,6 +198,93 @@ describe('Cascading Application', () => {
       expect(logger.info).toHaveBeenCalledWith('The branch test-cascading/1.0 is the last branch of the cascading. The process will stop.');
     });
 
+    describe('on onlyCascadeOnHighestMinors options', () => {
+
+      it('should ignore branches not to the latest minor when true', async () => {
+        customization.loadConfiguration = customization.loadConfiguration.mockResolvedValue({
+          ...DEFAULT_CONFIGURATION,
+          cascadingBranchesPattern: 'test-cascading/.*',
+          onlyCascadeOnHighestMinors: true,
+          ignoredPatterns: []
+        });
+        customization.getBranches = customization.getBranches.mockResolvedValue([
+          'test-cascading/1.0',
+          'test-cascading/1.1',
+          'test-cascading/1.2',
+          'test-cascading/2.0'
+        ]);
+        customization.isBranchAhead = customization.isBranchAhead.mockResolvedValue(true);
+        customization.createBranch = customization.createBranch.mockResolvedValue();
+        customization.getPullRequests = customization.getPullRequests.mockResolvedValue([]);
+        customization.createPullRequest = customization.createPullRequest.mockResolvedValue({
+          id: 1,
+          originBranchName: '',
+          isOpen: true,
+          mergeable: true,
+          body: render(mockBasicTemplate, { isConflicting: false, targetBranch: 'main', currentBranch: 'release/0.1', bypassReviewers: true }, { async: false })
```

Comment:
release/0.1 is not part of the list of supported branches line 212

Reply:
not a problem it is unused mock to unlock the process (it is the object to generate the PR template, so not the purpose of this test :))